src/common/cluster-ipc.ts

```diff
@@ -9,7 +9,7 @@ export const clusterIpc = {
       const cluster = clusterStore.getById(clusterId);
       if (cluster) {
         if (frameId) cluster.frameId = frameId; // save cluster's webFrame.routingId to be able to send push-updates
-        return cluster.activate(true);
+        return cluster.activate();
```

Comment:
While debugging IPC I noticed that activate really don't need a parameter. Idk why it was there 🤔

Reply:
Yes, indeed it's redundant if we put initial `disconnected` state to `true`